platform/config-core/src/main/java/org/motechproject/config/core/bootstrap/impl/BootstrapManagerImpl.java

```diff
@@ -62,8 +62,10 @@ public BootstrapConfig loadBootstrapConfig() {
 
     @Override
     public void saveBootstrapConfig(BootstrapConfig bootstrapConfig) {
+        configLocationFileStore.add(bootstrapConfig.getMotechDir());
```

Comment:
Is this correct? I would rather expect adding this with /config appended, rather than doing during the file check